src/lib/components/data-converter-port-settings.svelte

```diff
@@ -18,7 +19,7 @@
       on:keydown|stopPropagation
     />
     <p data-testid="data-encoder-info" class="text-sm w-[220px]">
-      *If both are set, the Codec Server endpoint will be used.
+      {translate('data-encoder', 'port-info')}
```

Comment:
Small thing, but wondering if we want to leave `*` out of the translation since it's more of a stylistic aspect.
```suggestion
      *{translate('data-encoder', 'port-info')}
```

Reply:
Good call, I removed it from the i18n string and used `before:content-['*']` to add the asterisk